spark3-extensions/src/test/java/org/apache/iceberg/spark/extensions/SparkRowLevelOperationsTestBase.java

```diff
@@ -48,32 +48,32 @@ public SparkRowLevelOperationsTestBase(String catalogName, String implementation
   @Parameters(name = "catalogName = {0}, implementation = {1}, config = {2}, format = {3}, vectorized = {4}")
   public static Object[][] parameters() {
     return new Object[][] {
-        { "testhive", SparkCatalog.class.getName(),
-            ImmutableMap.of(
-                "type", "hive",
-                "default-namespace", "default"
-            ),
-            "orc",
-            true
-        },
+//        { "testhive", SparkCatalog.class.getName(),
+//            ImmutableMap.of(
+//                "type", "hive",
+//                "default-namespace", "default"
```

Comment:
Used for local testing?

Reply:
Yes, will remove. Sorry about that, I usually look through the PR to catch these before review!